test_newsheadlines.py

```diff
@@ -0,0 +1,15 @@
+from newsheadlines import cli
+from click.testing import CliRunner
+
+
+class TestNews():
+    def test_cli_function():
+        runner = CliRunner()
+        result = runner.invoke(cli, ['1'])
+        url = url="https://newsapi.org/v2/top-headlines?sources=fox-news&apiKey=9b40cf73093e47e586cd61132f1d56fe&pageSize=10"
+        expected_output = [requests.get(url).json()['articles']]
```

Comment:
You are using the requests module but you are not importing it anywhere. It's not part of python